java/test/jmri/jmrit/logix/valid/IndicatorDemoTest.xml

```diff
@@ -407,15 +406,15 @@
           <path pathName="Main" blockName="OB2" />
         </toBlock>
       </portal>
-      <portal systemName="IP5" portalName="westYard-Main">
+      <portal systemName="IP:AUTO:0005" portalName="westYard-Main">
         <fromBlock blockName="OB2">
           <path pathName="westYard" blockName="OB2" />
         </fromBlock>
         <toBlock blockName="OB1Y">
           <path pathName="WestYard" blockName="OB1Y" />
         </toBlock>
       </portal>
-      <portal systemName="IP6" portalName="east-Main">
+      <portal systemName="IP:AUTO:0006" portalName="east-Main">
```

Comment:
This PR is changing the system names of the portals it reads, I.e. from IP6 to IP:AUTO:0006 and then stores that new name. Does this cause any trouble with later reading of the file?

Reply:
I too have concerns about this.  It isn’t when reading things in later that I see as an issue, but that the code is obviously invoking the automatic name generation routine when it is possible that the portal was given the system name by a use.

I don’t think the automatic name generation should ever be invoked when reading from a file.